File: 351003/Bitno/app/src/main/java/com/example/labrest/model/Creator.java
package com.example.labrest.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
@Data @NoArgsConstructor @AllArgsConstructor
public class Creator {
    private Long id;
    private String login;
    private String password;
    private String firstname;
    private String lastname;
}

File: 351001/Ushakov/lab3/publisher/internal/mapper/creator.go
package mapper

import (
	creator "github.com/Khmelov/Distcomp/351001/Ushakov/lab3/publisher/internal/model"
	"github.com/Khmelov/Distcomp/351001/Ushakov/lab3/publisher/internal/storage/model"
)

func MapCreatorToModel(cr creator.Creator) model.Creator {
	return model.Creator{
		ID:        cr.ID,
		Login:     cr.Login,
		Password:  cr.Password,
		FirstName: cr.FirstName,
		LastName:  cr.LastName,
	}
}

func MapModelToCreator(cr model.Creator) creator.Creator {
	return creator.Creator{
		ID:        cr.ID,
		Login:     cr.Login,
		Password:  cr.Password,
		FirstName: cr.FirstName,
		LastName:  cr.LastName,
	}
}
